### ISSUE TITLE
Auto televerse dont work.

### ISSUE THREAD

**Issue by stratege1401** [nextcloud/android#2098]
I have two Android unit. The first one is Asus zenpad s10 running Android 7.0.0. The second one is smartphone one+5  updated with the latest Oreo Android 8.
I've built my own nextcloud serveur based on Debian 9.3 stretch.
Using a W10 computer inside the local loop and Sync software i have no problems wifi automatique televerse.  my son which is living400 kms from my house is also using w10 computer with synch software without troubles.
 unfortunately both of my Android Unit are unable to automatic autoverse of any of the pictures from dcim/camera/download or any locations.

Using v2.0.0. From Google store on bot
 

### Environment data
One+5 Android version: Android Oreo 8.0.0 kernel 4.4.78-perf+ Oxygene 5.0.1 oneplusA5000_23_180103
Asus z500 s10 : P027 Android 7.0.0 kernel 3.18.35 build NRD90M

Unfortunately, I don't see my log ... And I don't know how to test the

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by mario** [comment]
Please try 3.0rc4 :)

**Comment by stratege1401** [comment]
3.orc4 on Asus same problem. Test on one5 in a few minutes. It is lunch time ! :)

**Comment by stratege1401** [comment]
Do I miss something ???...
Just tried rc3 on both of my Android ...
Absolutely no automatic televerse...
Did: move to automatic televerse. Choose the options as follow: ../photos/ not only wifi / no charge / use date folder...... Save......blue cloud on

But nothing happened.

Able to manually upload files/pics ... Able to synch

But no automatic televerse

**Comment by stratege1401** [comment]
Log from one+5:

[current_log.txt](https://github.com/nextcloud/android/files/1692807/current_log.txt)

**Comment by stratege1401** [comment]
Log from one+5

From test server, so don't care about info inside
content://com.microsoft.office.onenote.provider/8e45bdbe-cd55-47c0-83b0-16570b0b39bc

**Comment by stratege1401** [comment]
I am admin of this machine ... if i may help or create a test user for you, just ask.

**Comment by stratege1401** [comment]
test from local and ext network  using rc on my one+5.  Here log one+5 on 3/4G network ...
[currentLog.txt](https://github.com/nextcloud/android/files/1693072/currentLog.txt)

**Comment by stratege1401** [comment]
Did more test using my one+5.

If install date of the synch app is X, all pictures in dcim or elsewhere are not automatically synch if their creation date is >= X 

I manually upload all my pics. Clean the dcim folder. Reinstall the rc3. Shazam !!! New pictures taken after install are automatically synch.

**Comment by mario** [comment]
This is expected and not a bug.

**Comment by stratege1401** [comment]
well, i had to manually save 978 pictures, then clear my dcim space, then reinstall rc3, and shoot 1 picture, and another one to restart the euto-upload.

I'll expect some king of question/choose at first synch start, to included the "old" pictures.

Furthermore, this "feature" should be mention somewhere. So the actual normal behavior od this sync app wont be reported as a bug ... spend a few long minutes, manipulation, searching for a non-documented feature... or i am blind ?

**Comment by AndyScherzinger** [comment]
see #285 "old" pictures will be part of a future release when someone finds the time to implement it. So at the moment the installation time of the app etc doesn't matter it is rather that any picture taken _after_ auto upload has been activated for the respective folder will be uploaded but none before. As described in #285 it is planned to extend auto upload to also upload already existing files.

**Comment by stratege1401** [comment]
As autoteleverse is now fix in release android 3.0.0 , maybe you can consider closing this issue...

**Comment by AndyScherzinger** [comment]
Thanks for the feedback @stratege1401 ❤️
